docs/app/documentation/core-helpers/code-example/example-file.ts

```diff
@@ -0,0 +1,5 @@
+export interface ExampleFile {
+    name?: string;
+    code?: string;
+    language?: string;
```

Comment:
I'll make `code` mandatory. `language` isn't mandatory because if you don't provide it, it should be auto-detected. All examples do provide it, but it's not a mandatory thing